app/services/foreman_discovery/import_hooks/lldp_neighbor.rb

```diff
@@ -0,0 +1,11 @@
+# Automatically bonds interfaces based on lldp facts
+module ForemanDiscovery
+  module ImportHooks
+    class LldpNeighbor < ImportHook
+      def after_populate
+        return unless Setting[:discovery_auto_bond]
+        ::ForemanDiscovery::LldpNeighbors.eventually_make_bond(host)
```

Comment:
This could be moved into this class perhaps.

Reply:
I believe this definitely belongs here.

Reply:
Sure, we can move this in a separate PR.

Reply:
Timo 1 me 0. Okay :-)